components/Navigation.js

```diff
@@ -47,30 +49,39 @@ export default function Navigation({
       {!isNavigationOpen && (
         <>
           <StyledSearchMenu onClick={toggleSearchBar}>
-          <Image src={"/icons/search.png"} width={35}  height={35} alt= {"search icon"}/>
+            <Image
+              src={"/icons/search.png"}
+              width={35}
+              height={35}
+              alt={"search icon"}
```

Comment:
_Improving accessibility ✨_
```suggestion
             alt="Open search"
```